hyperspy/gui/egerton_quantification.py

```diff
@@ -456,3 +461,20 @@ def apply(self):
         self.interpolated_line = None
         self.reset_span_selector()
         self.find()
+
+
+# For creating a text handler in legend (to label derivative magnitude)
+class AnyObject(object):
```

Comment:
Cool!

Wouldn't something like `DerivativeTextParameters` and `DerivativeTextHandler` be more informative names for the classes?